File: SDKs/yodlee.com/1.1.0/python/src/sdk/models/shared/evaluateaddressrequest.py
import dataclasses
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import evaluateaccountaddress as shared_evaluateaccountaddress


@dataclass_json
@dataclasses.dataclass
class EvaluateAddressRequest:
    address: shared_evaluateaccountaddress.EvaluateAccountAddress = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('address') }})
    

File: SDKs/yodlee.com/1.1.0/typescript/src/sdk/models/shared/evaluateaddressrequest.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { EvaluateAccountAddress } from "./evaluateaccountaddress";



export class EvaluateAddressRequest extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=address" })
  address: EvaluateAccountAddress;
}
